public/app/features/explore/TableContainer.tsx

```diff
@@ -84,19 +124,25 @@ export class TableContainer extends PureComponent<Props> {
       }
     }
 
+    const label = this.renderLabel();
+
     return (
-      <Collapse label="Table" loading={loading} isOpen>
-        {dataFrame?.length ? (
+      <Collapse label={label} loading={loading} isOpen>
+        {dataFrame?.length && this.state.resultsStyle === TABLE_RESULTS_STYLE.table && (
           <Table
             ariaLabel={ariaLabel}
             data={dataFrame}
             width={tableWidth}
             height={height}
             onCellFilterAdded={onCellFilterAdded}
           />
-        ) : (
-          <MetaInfoText metaItems={[{ value: '0 series returned' }]} />
         )}
+
+        {dataFrame?.length && this.state.resultsStyle === TABLE_RESULTS_STYLE.raw && (
```

Comment:
Do we want this "raw" style only in Explore? Would it be useful in Dashboards?

Reply:
Right now (for prom at least) we don't display this table view for instant queries in dashboards:
<img width="1209" alt="image" src="https://user-images.githubusercontent.com/109082771/192781978-30be13bf-7d64-482c-b289-6ad29a1c3374.png">

Reply:
In Dashboards visualization has to be selected manually by the user (so the graph won't change to table just by changing the query). I was wondering if adding "raw" mode would make sense in Dashboard's table visualization? Or is it something we need only in Explore?
 
<img width="1535" alt="Screen Shot 2022-09-28 at 15 28 17" src="https://user-images.githubusercontent.com/745532/192791245-851e8cf5-5559-47b1-94b4-91971502677a.png">

Reply:
Oh yeah good catch, I would argue that since the point of this UI is for users to debug/play around with queries and isn't valuable to display metric results, that we should only introduce this option in the explore UI for now until users.